release/src/main/groovy/quickstart-java-apex.groovy

```diff
@@ -0,0 +1,47 @@
+#!groovy
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+t = new TestScripts(args)
+
+/*
+ * Run the direct quickstart from https://beam.apache.org/get-started/quickstart-java/
+ */
+
+t.describe 'Run Apache Beam Java SDK Quickstart - Apex'
+
+  t.intent 'Gets the WordCount Example Code'
+    QuickstartArchetype.generate(t)
+
+  t.intent 'Runs the WordCount Code with Apex runner'
+    // Run the workcount example with the apex runner
+    t.run "curl http://www.gutenberg.org/cache/epub/1128/pg1128.txt > /tmp/kinglear.txt"
```

Comment:
Quickstart uses the pom.xml, we should continue to use that or update the quickstart.